pom.xml

```diff
@@ -101,7 +101,7 @@
 
   <properties>
     <!--Dependency versions-->
-    <akka.version>2.5.1</akka.version>
+    <akka.version>2.5.6</akka.version>
     <akka.http.version>10.0.6</akka.http.version>
     <akka.persistence.cassandra.version>0.54</akka.persistence.cassandra.version>
     <akka.kryo.version>0.5.0</akka.kryo.version>
```

Comment:
You should review, merge and release this to avoid a version conflict:

https://github.com/ArpNetworking/commons/pull/61

Reply:
Delicious, thanks for the heads up.